### PR TITLE
Cleanup things around QuerySet()

### PR DESCRIPTION
**Pull request by hasegeli**

### CODE REVIEW

File: adminapi/dataset/filters.py [hunk 1]
```diff
@@ -137,45 +148,35 @@ def __repr__(self):
...
-    'between': Between,
-    'not': Not,
-    'startswith': Startswith,
-    'overlap': Overlap,
-    'insidenetwork': InsideNetwork,
-    'privateip': PrivateIP,
-    'publicip': PublicIP,
-    'optional': Optional,
-    'empty': Empty,
+    d.lower(): globals()[d] for d in dir() if not d.startswith('_')
```
Comment (TheFRedFox) [inline adminapi/dataset/filters.py]: Should we mention that somewhere, that all classes are collected automatically? Could be dangerous in future, if one does not know about that.

File: adminapi/utils/parse.py [hunk 2]
```diff
@@ -2,7 +2,7 @@ class ParseQueryError(Exception):
     pass
 
 
-def parse_function_string(args, strict=True):
+def parse_function_string(args, strict=True):   # NOQA: C901
```
Comment (TheFRedFox) [inline adminapi/utils/parse.py]: 3 spaces to the comment. 2 are enough. 😉

File: serveradmin/dataset/filters.py [hunk 3]
```diff
@@ -515,13 +480,23 @@ def from_obj(cls, obj):
...
+            )
+
         return _condition_sql(attribute, "{{0}} <<= ANY('{{{{{0}}}}}')".format(
             ','.join(str(n) for n in self.networks)
         ), servertypes)
 
 
 class InsideOnlyNetwork(InsideNetwork):
     def as_sql_expr(self, attribute, servertypes):
+        if attribute.type != 'inet':
```
Comment (TheFRedFox) [inline serveradmin/dataset/filters.py]: Mhm, somehow it does not feel elegant, if you need to duplicate code now with the changes.

File: serveradmin/resources/views.py [hunk 4]
```diff
@@ -15,7 +15,7 @@
 from serveradmin.serverdb.models import Project
 
 
-@login_required
+@login_required     # NOQA: C901
```
Comment (TheFRedFox) [inline serveradmin/resources/views.py]: Why so many spaces until the comment?

**Comment by TheFRedFox** [comment]
Just found some minor things. No code breaking things found.

File: serveradmin/resources/views.py (hunk 4 above)
Comment (hasegeli) [inline serveradmin/resources/views.py]: I align it to the next 4th.

File: adminapi/cmdline/query.py [hunk 5]
```diff
@@ -25,9 +25,11 @@ def main():
 
     attrs = options.attrs if options.attrs else ('hostname', )
```
Comment (hasegeli) [inline adminapi/cmdline/query.py]: Then it wouldn't be a `tuple`.  A `list` is more appropriate in here anyway, so I am changing it.

File: adminapi/dataset/filters.py (hunk 1 above)
Comment (hasegeli) [inline adminapi/dataset/filters.py]: I am adding a note about it.

File: adminapi/utils/parse.py (hunk 2 above)
Comment (hasegeli) [inline adminapi/utils/parse.py]: I align them to the next 4th column.

File: serveradmin/dataset/filters.py (hunk 3 above)
Comment (hasegeli) [inline serveradmin/dataset/filters.py]: I will clean it up again with the next one.  One of the purposes of this commit is to let the server-side filters go way completely.  Then we would have them all in a single place.

File: adminapi/cmdline/query.py (hunk 5 above)
Comment (TheFRedFox) [inline adminapi/cmdline/query.py]: `('hostname',)` would not be a tuple anymore? I'm talking about just the space, not the comma here. ;)

File: serveradmin/dataset/filters.py (hunk 3 above)
Comment (TheFRedFox) [inline serveradmin/dataset/filters.py]: Fair enough then.

**Pull request merged**